test/connectors/SubscriptionFieldsConnectorSpec.scala

```diff
@@ -52,6 +54,8 @@ class SubscriptionFieldsConnectorSpec extends UnitSpec with ScalaFutures with Mo
 
   def subscriptionFieldsBaseUrl(clientId: ClientId) = s"$urlPrefix/application/${clientId.value}"
 
+  def urlEncode(str: String, encoding: String = "UTF-8") = encode(str, encoding)
```

Comment:
Should we add urlEncode to apiContext and others as a way of ensuring it's done right?  Would save the need for .value.